codegen/build.gradle.kts

```
@@ -90,6 +90,10 @@ subprojects {
             useJUnitPlatform()
         }
 
+        tasks.withType(JavaExec) {
```

I think this is the groovy syntax. If you look at the stuff above and below they use something more like
```suggestion
        tasks.withType<JavaExec> {
```